.travis.yml

```diff
@@ -3,6 +3,7 @@ git:
   depth: 9999999
 jdk:
   - oraclejdk8
+dist: trusty
```

Comment:
Yes I searched online and it seems the "dist" if you don't specify it is now "xenial" (or something spelled similar). People started running into the problem because that "dist" didn't work with oraclejdk8. People either  tried openjdk8 or said they changed the dist to "trusty".

Reply:
![image](https://user-images.githubusercontent.com/29288945/61818673-541e4580-ae1f-11e9-86ee-c7c6ff430d3e.png)

Reply:
Got it.